### PR TITLE
Story/9774 replace cwa faq links

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Replacing CWA FAQ links

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9878

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1319,6 +1319,8 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 /* Home Info Card */
 
+"Home_AppInformationCard_Title" = "App-Information";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1322]: Why is this here? ^^

**Review by 30mar — CHANGES_REQUESTED** [review]
The title for the app information should be in German not english, i.e "App-Informationen"

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: Reverted back the string, it is now contained in Omar's PR.

**Review by 30mar — APPROVED** [review]

**Review by nickguendling — CHANGES_REQUESTED** [review]
I think you need to change it in all the other Localizable.links.strings files as well and use the English version there

**Review by CV113 — APPROVED** [review]
UA approved

**Review by CV113 — APPROVED** [review]
UA approved

**Review by nickguendling — APPROVED** [review]
